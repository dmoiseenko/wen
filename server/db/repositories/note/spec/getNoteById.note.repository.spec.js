const bootstrap = require('../../../bootstrap/bootstrap.js');
const { getNoteById } = require('../getNoteById.note.repository.js');
const { NoteNotFoundError } = require('../../../../utils/errors');


beforeEach(() => bootstrap());

it('should get note by id', async () => {
  const user = await getNoteById(1);

  expect(user).toMatchSnapshot();
});

it('should throw NoteNotFoundError if note wasn\'t found', async () => {
  await expect(getNoteById(100)).rejects.toBeInstanceOf(NoteNotFoundError);
});
