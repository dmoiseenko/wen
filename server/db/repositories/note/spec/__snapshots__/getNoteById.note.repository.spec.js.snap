// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get note by id 1`] = `
Object {
  "id": 1,
  "text": "Well done is better than well said. Benjamin Franklin",
}
`;
